info.portfolio_performance.PortfolioPerformance.json

```diff
@@ -0,0 +1,67 @@
+ {
+  "app-id": "info.portfolio_performance.PortfolioPerformance",
+  "runtime": "org.gnome.Platform",
+  "runtime-version": "3.38",
+  "sdk": "org.gnome.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk11"
+  ],
+  "command": "/app/jre/bin/PortfolioPerformance",
+  "finish-args": [
+    "--socket=x11",
+    "--socket=wayland",
```

Comment:
does this actually work under wayland natively? I thought java didn't have that working yet (xwayland still uses the x11 permissions)

Reply:
I just started this under Wayland and it works. However buttons looks strange. Does this mean `--socket=wayland` can stay or needs to be removed?

Reply:
Okay, buttons look better without the `--socket=wayland` thus I remove it for now.